active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,393 @@
+
+When a user interacts with the pager control and selects an edge button or page number, the page changed event will fire. 
+This event will provide the index the user is coming from (PreviousPageIndex) and the index the user is navigating to (NewPageIndex). 
+
+XAML
+```XAML
+<Page
+    x:Class="WindowsGallaryApp.Samples.DataGridSamplePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    Loaded="OnPageLoaded">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+        <controls:DataGrid x:Name="dataGrid" AutoGenerateColumns="True"/>
+        <muxc:ProgressRing x:Name="progressRing" Visibility="Collapsed" IsActive="false">
+        <muxc:Pager x:Name="pager" Grid.Row="1" PageChanged="OnPageChanged" PagerDisplayMode="NumberPanel"/>
+    </Grid>
+</Page>
+```
+C#
+```CSharp
+public sealed partial class DataGridSamplePage : Page
+{
+        private const int imagesPerPage = 25;
+
+        public DataGridSamplePage()
+        {
+                this.InitializeComponent();
+        }
+
+        private async void OnPageLoaded(object sender, RoutedEventArgs e)
+        {
+                pager.NumberOfPages = await ImageGenerator.GetNumberOfPages(imagesPerPage);
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(0, imagesPerPage);
+                progressRing.Visibility = "Collapsed";
+                progressRing.IsActive = false;
+        }
+        private async void OnPageChanged(Pager sender, PageChangedEventArgs args)
+        {
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(args.CurrentPage, imagesPerPage);
+                progressRing.Visibility = "Visible";
+                progressRing.IsActive = true; 
+        }
+}
+```
+
+# API Notes
+| Name | Description| Default | 
+|:---:|:---|:---|
+| PagerDisplayMode | Enum that contains 4 values (Auto, ComboBox, NumberBox, ButtonPanel) to represent the look of the pager control. When Auto is selected, the display mode will be ComboBox if the NumberOfPages property is less than 11 otherwise it will be NumberBox. | Auto |
+| PagerButtonVisibilityBehavior | Enum that contains 4 values (Visible, HiddenOnEdge, Hidden) that allows the app developer to hide or show the four edge buttons. HiddenOnEdge will remove the appropriate buttons if the selected page is the last or first page. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. When Auto is selected, the visibility mode will be AlwaysVisible. | Visible |
```

Comment:
There are only 3 values now, we removed `Auto`.

Reply:
Updated.